scheduler/src/cook/mesos/api.clj

```diff
@@ -1715,10 +1731,13 @@
                                :description "Reasons the job isn't being scheduled."}
                           400 {:description "Invalid request format."}
                           404 {:description "The UUID doesn't correspond to a job."}}}})))
-
     (ANY "/queue" []
          (waiting-jobs mesos-pending-jobs-fn is-authorized-fn))
     (ANY "/running" []
          (running-jobs conn is-authorized-fn))
     (ANY "/list" []
-         (list-resource (db conn) fid is-authorized-fn))))
+         (list-resource (db conn) fid is-authorized-fn)))
+   (format-params/wrap-restful-params {:formats [:json-kw]
```

Comment:
With compojure api, you can't turn off output processing without also disabling input processing. This is basically re-inserting the input processing they provide by default. It also lets us remove it from the liberator resources since they're now wrapped by this middleware.